docs/bots.md

```diff
@@ -54,3 +54,26 @@ Example of creating the bot user with the API.
 ```sh
 curl -X POST -H "content-type: application/json" -H "x-rems-api-key: 42" -H "x-rems-user-id: owner" http://localhost:3000/api/users/create --data '{"userid": "rejecter-bot", "name": "Rejecter Bot", "email": null}'
 ```
+
+## Bona Fide bot
+
+User id: `bonafide-bot`
```

Comment:
We use "bona fide" and "bonafide". I wonder if `bona-fide-bot` would make more sense?

Reply:
Yeah, it seems "bona fide" is more correct, even though "bonafide" is also seen. However I wonder if this should be called something like "referral-bot" or "researcher-status-bot"? I'm not sure we want to spread the bona fide term in the code since it isn't even used by the GA4GH standards (which talk about "researcher status").

Reply:
@jaakkocsc says let's go with "bona fide"

Reply:
Yes so with `bona-fide-bot` too then right?

Reply:
I'll rename everything :+1: